src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -105,4 +109,7 @@ class OPENVINO_GENAI_EXPORTS WhisperPipeline {
     WhisperGenerationConfig get_generation_config() const;
     void set_generation_config(const WhisperGenerationConfig& config);
 };
+
+OPENVINO_GENAI_EXPORTS std::pair<std::string, Any> chunk_streamer(ChunkStreamerVariant func);
+OPENVINO_GENAI_EXPORTS std::pair<std::string, Any> whisper_generation_config(const WhisperGenerationConfig& config);
```

Comment:
I think we can overload existing function like `streamer` and `generation_config` instead of introducing Whisper specific. 

Example:
https://github.com/openvinotoolkit/openvino.genai/blob/2370f6a9f171e9c47560622d630bca9b5b8b22c9/src/cpp/include/openvino/genai/image_generation/generation_config.hpp#L99-L100

Reply:
Great point, thanks, done.